Article/Article.less

```diff
@@ -42,9 +43,6 @@
 }
 
 .article-open {
-  height: 400px;
+  max-height: 400px;
```

Comment:
This is great, a lot of people missed this and it means their text was hidden on certain articles. Well done for spotting it!